ui/main/src/app/modules/monitoring/monitoring.component.html

```diff
@@ -28,7 +28,7 @@
 
      </div>
      <div  class="opfab-response-checkbox">
-        <label style="font-weight:bold; margin-left:5px; margin-bottom: 0px;" class="opfab-checkbox" translate>feed.filters.showCardsWithResponse.label
+        <label id="opfab-showCardsWithResponse" style="font-weight:bold; margin-left:5px; margin-bottom: 0px;" class="opfab-checkbox" translate>feed.filters.showCardsWithResponse.label
```

Comment:
our ids are with dashes between words, not uppercase letters. so rename opfab-showCardsWithResponse